libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFNetworkPlugin/SFNetworkPlugin.m

```diff
@@ -121,13 +121,13 @@ - (void) pgSendRequest:(CDVInvokedUrlCommand *) command
     __weak typeof(self) weakSelf = self;
     SFRestAPI *restApiInstance = doesNotRequireAuthentication ? [SFRestAPI sharedGlobalInstance] : [SFRestAPI sharedInstance];
     
-    [restApiInstance sendRESTRequest:request
-                                      failBlock:^(NSError *e, NSURLResponse *rawResponse) {
+    [restApiInstance sendRequest:request
+                                      failureBlock:^(id response, NSError *e, NSURLResponse *rawResponse) {
                                           __strong typeof(self) strongSelf = weakSelf;
                                           CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:e.localizedDescription];
```

Comment:
Similar comment to https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/3222/files#r443859170, should we send back the response somehow (may be in 9.0)?

Reply:
Yes, sounds good. cc: @bbirman